### PR TITLE
Txid tooltip fix

### PR DESCRIPTION
**Pull request by usselman**
As per this issue: https://github.com/pow-co/pow.co/issues/35
I have added a react-tooltip package and implemented it to show a link to an explanation of txids when hovering over 'txid' in the BoostContentCard

### CODE REVIEW

File: components/BoostContentCard.tsx [hunk 1]
```diff
@@ -137,16 +139,35 @@ const BoostContentCard = ({ content_txid, content_type, content_text, count, dif
                             </div>
                             )}
                             <div className='grow'/>
+                            
                             <a  onClick={(e:any)=>e.stopPropagation()}
                                 target="_blank"
                                 rel="noreferrer"
                                 href={`https://whatsonchain.com/tx/${content_txid}`}
                                 className="text-xs leading-5 whitespace-nowrap text-gray-500 dark:text-gray-300 hover:text-gray-700 hover:dark:text-gray-500"
+                                id="txid"
```
Comment (1aristotelis) [inline components/BoostContentCard.tsx]: an id is supposed to be a **unique** identifier in the DOM. You cannot have different elements with the same id named "txid".

I recommend using the actual txid as an id in that case :)

**Review by 1aristotelis — CHANGES_REQUESTED** [review]

File: components/BoostContentCard.tsx (hunk 1 above)
Comment (derrickhorton) [reply components/BoostContentCard.tsx]: Great catch!

Comment (usselman) [reply components/BoostContentCard.tsx]: I implemented the id in the way you recommended but now I'm seeing that only some txids get the tooltip and not others, any ideas?

It is not clear which show tooltips and which don't.

Screen Shot 2023-02-28 at 1 02 28 AM
Screen Shot 2023-02-28 at 1 02 15 AM
Screen Shot 2023-02-28 at 1 02 08 AM

Comment (1aristotelis) [reply components/BoostContentCard.tsx]: might be that `content_txid` does not exist on that specific post (could be named `txid` or `transaction`)

Comment (usselman) [reply components/BoostContentCard.tsx]: fixed!

**Pull request merged**